int32   id
BoundingBox bbox